src/context/WalletProvider/local-wallet.ts

```diff
@@ -0,0 +1,39 @@
+import { KeyManager } from './config'
+/**
+ * Defining storage up here,
+ * so it can be changed to other storage types easily
+ */
+const storage = {
+  set: (key: string, value: string) => localStorage.setItem(key, value),
+  get: (key: string) => localStorage.getItem(key),
+  remove: (key: string) => localStorage.removeItem(key)
```

Comment:
why not just leverage the same `localforage` module that was added for `redux-persist`? I don't even see where the variable `localStorage` is defined.

Reply:
`localStorage` is coming from the global `Window` object in browser environments so it doesn't need to be explicitly imported.
Agreed we should probably leverage `localforage` usage, this is effectively duplicating the storage logic.